Added simple tests to confirm split jaws records work correctly

ThomasLohnert opened the pull request:
Added tests for https://github.com/ISISComputingGroup/EPICS-jaws/pull/8

DominicOram commented on test_config/motor_test/jaws.cmd:
```diff
@@ -0,0 +1,3 @@
+$(IFIOC_GALIL_01=#) dbLoadRecords("$(JAWS)/db/split_jaws_header_vertical.db","P=$(MYPVPREFIX)MOT:,JAWS=JAWS1:,mXN=MTR0101,mXS=MTR0102,mXW=MTR0202,mXE=MTR0201")
```
If you look at how we test the `motorExtensions` we put the test settings files into the `motorExtensions` directory, I think putting this in the `jaws` directory might be better.

ThomasLohnert replied:
Done. Added folders for each type of jawset in `jaws\settings\` containing a `.cmd` each and updated the path in `vertical_jaws.py` IOC tests to match.

DominicOram commented on tests/split_jaws.py:
```diff
@@ -0,0 +1,129 @@
+            self.ca.set_pv_value("{}.VMAX".format(axis), 100)
+            self.ca.set_pv_value("{}.VELO".format(axis), 100)
+
+    def test_GIVEN_ioc_started_THEN_north_underlying_mtr_fields_can_be_read(self):
```
The first four tests can easily be paramterized.

ThomasLohnert replied:
With the latest changes, there are 2 test classes which both run the same set of tests for regular and split jaws respectively. With parameterized tests, I could not overwrite the underlying motor PV addresses in the split jaws test class, so I have opted against doing this.

ThomasLohnert commented:
- Refactored tests so that the same set is run against jawsets with different setups of their underlying motors. 
- Added tests for ABLE and LOCK status.
- Could not reproduce the failure, let me know if you still see it and we can investigate.

DominicOram commented:
Thanks for the refactor and the additional tests.

I fixed the issue on my machine by setting the gaps to zero on setUp, the underlying issue is https://github.com/ISISComputingGroup/IBEX/issues/4645
